reconstructSPI/iterative_refinement/expectation_maximization.py

```diff
@@ -130,6 +128,14 @@ def iterative_refinement(self, wiener_small_number=0.01, count_norm_const=1):
             .reshape(map_shape)
         )
 
+        wiener_small_numbers = IterativeRefinement.compute_ssnr(
+            self.particles, ctfs, 0.01
+        )
+        wiener_small_numbers = np.where(
+            wiener_small_numbers == 0, 0.1, wiener_small_numbers
+        )
+        wiener_small_numbers = 1 / wiener_small_numbers
+
```

Comment:
I would write a little wrapper for this, and include a test for it. 
* shape
* greater or less than some number

we want to keep `iterative_refinement` as focused as possible, and encapsulate things inside into modular functions which we can test.